### PR TITLE
Civ 4135

### PR DESCRIPTION
**Pull request by pliao-hmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-4135

### Change description ###
CoS claim details, front end screen shots are in the ticket
merged with following tickets
https://tools.hmcts.net/jira/browse/CIV-4113
https://tools.hmcts.net/jira/browse/CIV-3456

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseField/CaseField-COS-nonprod.json [hunk 1]
```diff
@@ -48,34 +97,12 @@
...
-    "SecurityClassification": "Public"
-  },
   {
     "CaseTypeID": "CIVIL",
     "ID": "cosLabelForClaimant1_2",
     "Label": "Claimant's name <br /> ${applicant1.partyName}",
     "FieldType": "Label",
     "SecurityClassification": "Public"
-  },
-  {
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseField/CaseField-COS-nonprod.json:65]: Deleting these varialbe will affect CoS notify screen?

File: ccd-definition/CaseField/CaseField-COS-nonprod.json [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant2_1",
+    "Label": "Claimant's name <br /> ${applicant1.partyName}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseField/CaseField-COS-nonprod.json:37]: I would suggest using the variable I added "${respondent1.partyName}" as we shouldnot be depending on variables defined specific for other tab not related to CoS.

File: ccd-definition/CaseField/CaseField-COS-nonprod.json (hunk 1 above)
Comment (pliao-hmcts) [reply ccd-definition/CaseField/CaseField-COS-nonprod.json:65]: these fields are moved up to line 25/32

File: ccd-definition/CaseField/CaseField-COS-nonprod.json (hunk 2 above)
Comment (pliao-hmcts) [reply ccd-definition/CaseField/CaseField-COS-nonprod.json:37]: ${respondent1.partyName} and other respondent1 values are not loaded at CoS notify detail page so have to use this value

Comment (pliao-hmcts) [reply ccd-definition/CaseField/CaseField-COS-nonprod.json:37]: fixed

File: ccd-definition/CaseField/CaseField-COS-nonprod.json [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+    "CaseTypeID": "CIVIL",
+    "ID": "cosNotifyClaimDetails2",
+    "Label": " ",
+    "FieldType": "NotifyClaimDetailsCoS",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant1",
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseField/CaseField-COS-nonprod.json:18]: Is it possible to use same label "cosLabelForClaimant1" for both places in notify claim and notify claim details? Two labels for showing same date.

Comment (pliao-hmcts) [reply ccd-definition/CaseField/CaseField-COS-nonprod.json:18]: move to tab task

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**